pkg/tektonlog/tektonlog.go

```diff
@@ -304,6 +292,57 @@ func (t *TektonLogger) getRunningBuildLogs(ctx context.Context, pa *v1.PipelineA
 	return nil
 }
 
+func (t *TektonLogger) collectStages(ctx context.Context, pipelineRuns []*tektonapis.PipelineRun) []stageTime {
+	var podTimes []stageTime
+	for _, prInitial := range pipelineRuns {
+		//we need fresh pipeline to be able consume newly executed tasks/pods
+		pr, _ := t.TektonClient.TektonV1beta1().PipelineRuns(t.Namespace).Get(ctx, prInitial.Name, metav1.GetOptions{})
```

Comment:
can you please check the err returned here? otherwise in case of error, you'll have a nil `pr` that will break at the next line

Reply:
done